selfservice/hook/session_issuer.go

```diff
@@ -78,6 +79,13 @@ func (e *SessionIssuer) executePostRegistrationPostPersistHook(w http.ResponseWr
 		return err
 	}
 
+	trace.SpanFromContext(r.Context()).AddEvent(events.NewLoginSucceeded(r.Context(), &events.LoginSucceededOpts{
```

Comment:
This now only triggers for the browser flow, for the API flow the event is missing